dspace/config/emails/subscriptions_content

```diff
@@ -0,0 +1,15 @@
+## E-mail sent to designated address about updates on subscribed items
+##
+## Parameters: {0} Collections updates
+##             {1} Communities updates
+
```

Comment:
This Email has no subject.  Please add a Subject... the old `subscription` email had this one:
```
#set($subject = 'DSpace Subscription')
```

Reply:
Subject added